packages/atomic/src/components/common/atomic-icon/atomic-icon.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import bgIcons from '@salesforce-ux/design-system/design-tokens/dist/bg-standard.common';
 import {html} from 'lit-html';
-import defaultStory from '../../../../.storybook/default-story';
+import defaultStory from '../../../../../../utils/atomic-storybook/.storybook/default-story';
```

Comment:
Yeah, I would like if you could check if those imports could be simplified with a TS config change.

If it's too complicated and start randomly breaking other things, then so be it.

But the import chain is a bit extreme right now 😉